### DISCUSSION TITLE
proposal（Heads）

### DISCUSSION THREAD

**Discussion by 1425115471** [Zrips/CMI#1814] [Enhancement]
Can be set [LOOT_BONUS_MOBS] Grade
increase head Falling
Increased probability of each level of predation

  horse:
    DropChance: 0.01
    LowerWithEachDrop: 0
    plunder: 1

This means that each level of plunder increases the drop rate by 1%.
My English is not very good, I'm sorry.

**Comment by Zrips** [comment]
Plunder? Whats that?

**Comment by 1425115471** [comment]
LOOT_BONUS_MOBS（enchanting）

**Comment by 1425115471** [comment]
enchant
